app/javascript/pages/home.js

```diff
@@ -15,9 +18,47 @@ const styles = {
     fontSize: '50px'
   }
 };
+
 const Home = () => {
   const navigate = useNavigate();
   const [articles, setArticles] = useState([]);
+  const [isModalOpen, setModalOpen] = useState(false);
+  const [data, setData] = useState({
+    title: '',
+    description: '',
+    creator: ''
+  });
+
+  const handleModalToggle = () => {
+    setModalOpen(!isModalOpen);
+  };
+
+  const handleSubmit = (e) => {
```

Comment:
@Ron-Lavi to move these stuffs to `ArticleFrom` component, we need to add component on the `onClick` so that it'll render that component. This is what you want me to do implement here?

Reply:
Hi, so the modal toggle will remain on the `home.js` component,
but the modal related things will remain in the `ArticleForm` component